src/pybind/mgr/influx/module.py

```diff
@@ -108,7 +109,46 @@ def get_df_stats(self):
                     }
                 }
                 data.append(point)
-        return data
+                pool_info.update({str(pool['id']):pool['name']})
+        return data, pool_info
+
+    def get_pg_summary(self, pool_info):
+        osd_sum = self.get('pg_summary')['by_osd']
```

Comment:
Suggest you do a single self._get('pg_summary') and then re-use the result.  the `get` calls aren't hugely expensive, but let's avoid calling it twice in two lines.